src/aggregate.d

```diff
@@ -126,7 +126,7 @@ public:
             sc2.inunion = 1;
         sc2.protection = Prot(PROTpublic);
         sc2.explicitProtection = 0;
-        sc2.structalign = STRUCTALIGN_DEFAULT;
+        sc2.aligndecl = null;
```

Comment:
And, by renaming it, I easily found the places I should fix. That's not noise.
If you want to keep existing name `structalign`, please give me more strong reason.